### ISSUE TITLE
Enable multiple default tagging per feed

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#2063]
Enable multiple default tagging per feed

**Comment by yannw** [comment]
Just wanted to open the same Issue.
-> +1 (:

**Comment by github-germ** [comment]
any chance multiple tags for a feed will be implemented soon? if not, is the prevailing solution to write a PyMISP API client to step through the feed after it's ingested?

(seems to be a dup of #1601)

**Comment by adulau** [comment]
Indeed, it's in the long list of features to do. We accept pull-request if this is utterly urgent for you ;-)

**Comment by imendiguren** [comment]
Hi, any idea how far is this issue from implementation?

**Comment by github-germ** [comment]
We've not used Tag Collections yet, but were wondering if there is a way to use a pre-defined Tag Collection as the Default Tag for a Feed? Something like adding a `tag_collection_id` column to the `feeds` table, provide that on `feeds/edit`...

**Comment by k7leafclover** [comment]
Bumping this. Ideally multiple default tags. If it would be easier to implement, allow tag collections to be selected here. Define tag collection > assign tag collection to feed.
